src/cim/components/ClusterDeployment/types.ts

```diff
@@ -64,7 +64,13 @@ export type ClusterDeploymentHostsSelectionValues = {
   autoSelectedHostIds: string[];
 };
 export type ClusterDeploymentHostsDiscoveryValues = {
-  /* TODO(mlibra): CNV, OCS */
+  autoSelectHosts: boolean;
+  hostCount: number;
+  useMastersAsWorkers: boolean;
+  agentLabels: string[];
+  locations: string[];
+  selectedHostIds: string[];
+  autoSelectedHostIds: string[];
```

Comment:
```suggestion
  useMastersAsWorkers: boolean;
  selectedHostIds: string[];
```

a lot of these are not used, so we should remove them

Reply:
done!